src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -120,6 +120,13 @@ public enum SlackDistributionFailureBehavior {
 
     protected static final double GENERATOR_VOLTAGE_CONTROL_MIN_NOMINAL_VOLTAGE_DEFAULT_VALUE = -1d;
 
+    public enum FictitiousGeneratorVoltageControlMode {
```

Comment:
You mean instead of a boolean parameter ? No "need". I just saw the fictitiousGeneratorVoltageControlMode as a string param with values ALWAYS , NORMAL. ALWAYS meaning Always maintaining voltage, and NORMAL the normal behaviour - that is maintaining Voltage in started and target P between pmin and pmax.